addons/armaos/functions/fnc_os_tree.sqf

```diff
@@ -0,0 +1,103 @@
+/**
+ * Prints the filesystem tree with the current working directory as root to stadout.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Options <[STRING]>
+ * 3: Command Name <STRING>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer", "_options", "_commandName"];
+
+private _commandOpts = [];
+private _commandSyntax =
+[
+	[
+			["command", _commandName, true, false]
+	]
+];
+private _commandSettings = [_commandName, _commandOpts, _commandSyntax];
+
+[] params ([_computer, _options, _commandSettings] call AE3_armaos_fnc_shell_getOpts);
+
+if (!_ae3OptsSuccess) exitWith {};
+
+private _pointer = _computer getVariable "AE3_filepointer";
+private _filesystem = _computer getVariable "AE3_filesystem";
+
+private _current = [_pointer, _filesystem] call AE3_filesystem_fnc_resolvePntr;
+
+private _terminal = _computer getVariable "AE3_terminal";
+private _user = _terminal get "AE3_terminalLoginUser";
+
+private _level = 0;
+
+private _fnc_tree_recursive =
```

Comment:
Subfunction should be defined at the top of the function for consistency

Reply:
Ok. Will do that.